Documentation/guides/building-apps/build-items.md

```diff
@@ -19,6 +19,38 @@ or library project is built.
 Supports [Android Assets](https://developer.android.com/guide/topics/resources/providing-resources#OriginalFiles),
 files that would be included in the `assets` folder in a Java Android project.
 
+The `AndroidAsset` ItemGroup also supports additional metadata for generating [Asset Packs](https://developer.android.com/guide/playcore/asset-delivery). Adding the `AssetPack` attribute to and `AndroidAsset` will automatically generate an asset pack of that name. This feature is only supported when using the `.aab`
+`AndroidPackageFormat`. The following example will place `movie2.mp4` and `movie3.mp4` in separate asset packs.
```

Comment:
I'm now wondering what `$(OutputPath)` looks like.  Are all the asset packs *within* the `.aab` file?  Or are there *multiple* output files within `$(OutputPath)` now?  If the latter, what are they named?

Reply:
So the intermediate "assetpack" zip files do NOT end up in the `$(OutputPath)`. I didn't want to confuse matters by poluting the `$(OutputPath)` with those. So they stay in the `obj` directory. 

ALL the assetpacks will end up being folded into the `.aab` file which ends up in `$(OutputPath)`.